doc/source/dask-on-ray.rst

```diff
@@ -78,6 +77,32 @@ Why use Dask on Ray?
 
 Dask-on-Ray is an ongoing project and is not expected to achieve the same performance as using Ray directly. All `Dask abstractions <https://docs.dask.org/en/latest/user-interfaces.html>`__ should run seamlessly on top of Ray using this scheduler, so if you find that one of these abstractions doesn't run on Ray, please `open an issue <https://github.com/ray-project/ray/issues/new/choose>`__.
 
+Best Practice for Large Scale workloads
+---------------------------------------
+For Ray 1.3, the default scheduling policy is to pack tasks to the same node.
+It is more desirable to load balance tasks if you run a large scale / memory intensive Dask on Ray workloads.
+
+In this case, there are two recommended setup.
+- Setting an internal config flag `scheduler_loadbalance_spillback` to change the scheduler to load balance tasks. 
+- Setting the head node's `num-cpus` to 0 so that tasks are not scheduled on a head node.
+
+Single Node Example
+===================
+If you start a cluster using `ray.init`, it means you only have a head node. So don't specify `num_cpus=0`.
+
+.. code-block:: python
+
+  ray.init(_system_config={"scheduler_loadbalance_spillback": True})
+
+Multi Node Example
+==================
+If you are using `ray start`, start a head node with these parameters.
+
+.. code-block:: bash
+
+  ray start --head --num-cpus=0 --system-config='{"scheduler_loadbalance_spillback":true}'
```

Comment:
Can we use `RAY_SCHEDULER_LOADBALANCE_SPILLBACK=1` env var so they don't have to modify system config for this?